### DISCUSSION TITLE
Hardware that support HDMI 2.1/4K@120Hz

### DISCUSSION THREAD

**Discussion by Titanos** [awawa-dev/HyperHDR#653] [Q&A]
I have at the time a running HyperHDR setup that uses the windows screen grabber method. I am however planning to go the capture card route in the near future and have tried to work out how to make a complete setup that allows for HDMI 2.1 4K@120Hz with all the bells and whistles.

The idea is to connect a HTPC (with a HDMI 2.1 GPU), PS5 and a Google chrome with google TV to this setup.

Is it possible to connect all the hardware to a HDMI switch like this one:

https://www.amazon.se/gp/product/B09QRZT92N/ref=ox_sc_act_title_2?smid=A9MU9JXBQM05V&th=1

And from the HDMI out on that switch connect to:

https://www.amazon.se/gp/product/B0C739QQ4Q/ref=ox_sc_act_title_1?smid=A9MU9JXBQM05V&psc=1

And presumably set the splitter to downscale to 1080P@120Hz and from HDMI OUT2 connect to a MS2130 .
The MS2130 would then be connected to either a Raspberry Pi 5 or my current HTPC where HyperHDR is installed.

Is this setup possible and would it work with all the possible refresh rates within HDMI 2.1 (up to 144Hz i believe from the PC), HDR, HDR+ and Dolby vision?

And if not, are there other solutions?

**Comment by dbell669** [comment]
you could try using The Avermedia 2.1 capture card. it can support 4k@60Hz for recording, but has 4k@144 passthrough. I dont think there are any external capture cards out at the moment that support 4k@120 recording, let alone 4k@144. I know Asus is releasing one as well, but I haven't heard any details about it so far.
